apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestAudioActivity.java

```diff
@@ -604,7 +604,7 @@ public void openAudio() throws IOException {
      * @param usage usage for the stream
      * @param contentType tupe of the stream
      */
-    protected void setStreamControlByAttributes(int usage, int contentType) {
+    protected void maybeSetStreamControlByAttributes(int usage, int contentType) {
```

Comment:
This is the method that sets it (assuming the API is implemented).

Reply:
See newest iteration. I added a helper function